web/templates/homepage/sidebar.html.eex

```diff
@@ -1,29 +1,31 @@
 <div class="filter-sidebar">
   <div class="fixed sidebar top-0 bg-white bottom-0 w-80 w-70-ns w-50-m w-30-l pa3 overflow-y-scroll tl shadow-2">
-    <div>
+    <div class="tl lm-grey w-100 tr mb4">
+      <%= label :filter, :filter, class: "f2 pointer" do
+        raw("<i class=\"fa fa-times\" aria-hidden=\"true\"></i>")
```

Comment:
`raw ~s(<i class="fa fa-times" aria-hidden="true"></i>)`

Would be my preference here (not an issue though)